### PR TITLE
ref/runner: clarify `--cloud-aws-security-group`

### PR DESCRIPTION
**Pull request by casperdcl**
- ref/runner: clarify `--cloud-aws-security-group`
- ~`npm update`~

### CODE REVIEW

**Review by 0x2b3bfa0 — APPROVED** [review]

**Comment by casperdcl** [comment]
btw some failing `yarn lint-ts` tests /CC @julieg18 probably due to previously missed bugs picked up after `npm update`?

File: package.json [hunk 1]
```diff
@@ -46,10 +46,10 @@
     "classnames": "^2.2.6",
     "color": "^3.1.2",
     "compression": "^1.7.4",
-    "date-fns": "^2.11.1",
+    "date-fns": "^2.28.0",
```
Comment (julieg18) [inline package.json:49]: Have you done research on these changes? cml.dev is still at Gatsby 2 so we probably should be more cautous when we're upgrading packages. We really should get cml.dev to Gatsby 4... I'll start working on that after I finish my current P1.

**Review by julieg18 — COMMENTED** [review]
Did you use npm instead of yarn to install? 

image

File: package.json (hunk 1 above)
Comment (casperdcl) [reply package.json:49]: argh srry yes was uning `npm` instead of `yarn` and for once missed the double lockfile warning :facepalm:

**Pull request merged**